lib/repo/patch_type.ex

```diff
@@ -1,6 +1,8 @@
 defmodule ExAudit.Type.Patch do
   use Ecto.Type
 
+  @type t() :: map()
```

Comment:
That type is never a map, and the type is also specified here: `ExAudit.Diff.changes`